app/views/content/landing/grow/_promo.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section( classes: %w[ml-background]) %>
```

Comment:
@garethwilliamsdesign As discussed, would be good to change this `ml-background` image to be more engaging (campaign image)

However, this can be done separately to this PR as it will affect all mailing list promos!